src/apps/external_executor/protobuf/historical.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+import "kv.proto";
+
+package externalexecutor.protobuf;
+
+option optimize_for = LITE_RUNTIME;
+
+service Historical
+{
+  // Historical data look up
+  rpc GetHistoricalData(HistoricalData) returns (QueryResponse) {}
+}
+
+message HistoricalData
+{
+  string map_name = 1;
+  bytes key = 2;
+  int64 view = 3;
+  int64 seqno = 4;
+}
```

Comment:
`view` and `seqno` should be stored as `uint64`, to match the C++ types.

It might be helpful to pull the view and seqno pair into a separate `TxID` type, to distinguish them/make it clear they're a closely-related pair? The latter is a fuzzy suggestion, unclear benefit.

```suggestion
message TxID
{
  uint64 view = 1;
  uint64 seqno = 2;
}

message HistoricalData
{
  string map_name = 1;
  bytes key = 2;
  TxID tx_id = 3;
}
```